.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -134,3 +135,4 @@ If the build fails after tagging:
 2. check if the fixes changed any extra crates, and do the required version increments
 3. update `README.md` with a **new** git tag
 4. tag a **new** release
+5. Update CHANGELOG.md
```

Comment:
Similarly, let's update the changelog as the 3rd step, so it is included in the tag.